admin/model/billmate/payment/modificator.php

```diff
@@ -0,0 +1,396 @@
+                                                if (stripos($line, $search) !== false) {
+                                                    // If indexes are not used then just set the found status to true.
+                                                    if (!$indexes) {
+                                                        $match = true;
+                                                    } elseif (in_array($i, $indexes)) {
+                                                        $match = true;
+                                                    }
+
+                                                    $i++;
+                                                }
+
+                                                // Now for replacing or adding to the matched elements
+                                                if ($match) {
+                                                    switch ($position) {
+                                                        default:
+                                                        case 'replace':
+                                                            $new_lines = explode("\n", $add);
+
+                                                            if ($offset < 0) {
+                                                                array_splice($lines, $line_id + $offset, abs($offset) + 1, array(str_replace($search, $add, $line)));
+
+                                                                $line_id -= $offset;
+                                                            } else {
+                                                                array_splice($lines, $line_id, $offset + 1, array(str_replace($search, $add, $line)));
+                                                            }
+                                                            break;
+                                                        case 'before':
+                                                            $new_lines = explode("\n", $add);
+
+                                                            array_splice($lines, $line_id - $offset, 0, $new_lines);
+
+                                                            $line_id += count($new_lines);
+                                                            break;
+                                                        case 'after':
+                                                            $new_lines = explode("\n", $add);
+
+                                                            array_splice($lines, ($line_id + 1) + $offset, 0, $new_lines);
+
+                                                            $line_id += count($new_lines);
+                                                            break;
+                                                    }
+                                                }
+                                            }
+
+                                            $modification[$key] = implode("\n", $lines);
```

Comment:
Use single quotes.

`$modification[$key] = implode('\n', $lines);`

Reply:
It will not work with single quotes.